user_flair/hero_flair.scss

```diff
@@ -626,4 +626,12 @@
 
 .flair-master-probius::before {
   background-position: 40px -2880px
+}
+
+.flair-genji::before {
+  background-position: 0 -2900px
+}
+
+.flair-master-genji::before {
```

Comment:
Since we don't have the master portrait flair yet, we should hold off on adding this rule.